apps/mudita-center-e2e/src/specs/help/contact-support-happy-path.ts

```diff
@@ -0,0 +1,206 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import NavigationTabs from "../../page-objects/tabs.page"
+import HomePage from "../../page-objects/home.page"
+import { sleep } from "../../helpers/sleep.helper"
+import HelpModalPage from "../../page-objects/help-modal.page"
+import HelpPage from "../../page-objects/help.page"
+import ContactSupportSuccessModalPage from "../../page-objects/contact-support-success-modal.page"
+import testsHelper from "../../helpers/tests.helper"
+import dns from "node:dns"
+
+describe("Mock Using Contact Support Form", () => {
+  before(async () => {
+    dns.setDefaultResultOrder("ipv4first")
+    const notNowButton = await HomePage.notNowButton
+    await notNowButton.waitForDisplayed()
+    await notNowButton.click()
+    await sleep(20000)
```

Comment:
if we change line 19: await notNowButton.waitForDisplayed() to await notNowButton.waitForDisplayed({ timeout: 20000 }) 
and if we remove line 21: await sleep(20000)

then test takes ~ 5 seconds instead of ~23seconds

Reply:
Fixed in [5500a6b](https://github.com/mudita/mudita-center/pull/2172/commits/5500a6bd67895a83a8d58fb765ea57d38cb8bc94)